packages/vhd-lib/Vhd/VhdDirectory.js

```diff
@@ -275,9 +275,15 @@ exports.VhdDirectory = class VhdDirectory extends VhdAbstract {
     const compressionType = this.compressionType
     const path = this._path + '/chunk-filters.json'
     if (compressionType === undefined) {
-      await this._handler.unlink(path)
+      try {
+        await this._handler.unlink(path)
+      } catch (error) {
+        if (error.code === 'ENOENT') {
+          throw error
+        }
```

Comment:
IIRC, I asked you about letting `Handler#unlink` deal with this error, and i don't remember you answering.

Unfortunately I don't remember in which PR :thinking:

Reply:
here : https://github.com/vatesfr/xen-orchestra/pull/6220#discussion_r867360552
and when searching through the code to check for impact, I found this

Reply:
I'll do a PR to handle this error by default.

Reply:
Already handled.